applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -430,7 +431,7 @@ static int bis_headset_cleanup(void)
 	return 0;
 }
 
-static int change_active_audio_stream(void)
+int broadcast_sink_change_active_audio_stream(void)
```

Comment:
Ref. comment above. "stream" is a bit indescriptive.

Reply:
Same comment as the one above:
Do you have any name suggestions? I cant come up with any better names that are not too long.
Added more explanation to the Doxygen doc.